composer.json

```diff
@@ -13,21 +13,19 @@
         }
     ],
     "require": {
-        "php": "^5.5|^7.0",
+        "php": "^7.2",
         "doctrine/common": "~2.0",
```

Comment:
Common is deprecated and splitted into several libraries, we should drop it.

Reply:
Hmm, true. Will check where is used. Probably in a separate pr